Changing instances

JustBerkhout opened the pull request:
I fleshed out a V1.0 for this tutorial. 

The main purpose of the tutorial is to highlight instance lifecycle events (reboot, resize, rebuild and delete) using the dashboard. 

This tutorials is part of the "Cloud Starter" series

markendr commented:
Looks good, a couple of typos
p1: "get you instance" should be "your"
p3: "takes les than" should be "less", also "sizes to other" should be "others".
Also "Managing Instances" might be a better title?

JustBerkhout commented:
- Fixed the typos (3x). 
- Fixed the order of the Title to be consistent  (Rebooting, Resizing, Rebuilding, Deleting).
- left the name as "changing-instances". 
It's not perfect, but Managing sounds more generic to me. I'm trying to convey: `making-changes-to-instances`, but thought of that title as to long/unwieldy.

JustBerkhout commented:
Adding " Making changes to existing instances. "in the frontmatter summary, to include it on the tutorial badge
